go-zen-chu/aictl
# Add AI Review comment to PR

## Why

- I want to add review result to each PR. aictl would be able to that now.

## What

- Add dockerfile to make it possible to write to /github/* dirs.
- Add tests

## QA, Evidence

- [x] In this PR, aictl will write a review for me

yay, please see this comment: https://github.com/go-zen-chu/aictl/pull/8#issuecomment-2379258562
